webapi/src/main/scala/org/knora/webapi/routing/admin/ProjectsRouteADM.scala

```diff
@@ -353,7 +381,35 @@ class ProjectsRouteADM(routeData: KnoraRouteData)
     }
 
   /**
-   * returns all members part of a project identified through shortname
+   * Returns all members part of a project identified through UUID
```

Comment:
```suggestion
   * Returns all members of a project identified through the project's UUID
```

Reply:
This goes for all the other docstrings here that are similar as well. I won't repeat it each time.